modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java

```diff
@@ -230,42 +229,6 @@ public void testGetLocalizationXmlFromPreferencesWithEmptyPreferences() {
 					"\">testValue</test>"));
 	}
 
-	@Test
-	public void testGetModifiedLocales() throws Exception {
-		String key = RandomTestUtil.randomString();
-
-		String defaultLanguageId = LocaleUtil.toLanguageId(
-			LocaleUtil.getDefault());
```

Comment:
@julschong Can you move the test to where you inline the code? I do not want to lost test coverage if possible

Tina.

Reply:
@tinatian Hi Tina, as our discussion, this was inlined into archived. we will not be adding the unit test

Reply:
@julschong Please update comment